### PR TITLE
Add basic CircleCI config

### PR DESCRIPTION
**Pull request by jonallured**
This PR adds CircleCI config plus the badge to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# Artsy CLI
+# Artsy CLI [![CircleCI][badge]][circleci]
+
+[badge]: https://circleci.com/gh/artsy/artsy-cli.svg?style=svg
+[circleci]: https://circleci.com/gh/artsy/artsy-cli
```
Comment (anandaroop) [inline README.md]: 😎

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.2 :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.2 :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.2 :rocket:

**Pull request merged**